palo_alto_pan_os/komand_palo_alto_pan_os/util/pan_os_requests.py

```diff
@@ -174,6 +174,14 @@ def get_output_with_exceptions(response):
 
         if output.get('response', {}).get('@status') == 'error':
             error = output['response']['msg']
+            line_error = error.get('line')
+            if line_error and [s for s in line_error if 'is invalid' or 'config validity' in s]:
```

Comment:
```suggestion
            if line_error and [s for s in line_error if 'is invalid' or 'config validity' in s]:
```
Shouldn't this be:
`if line_error and [s for s in line_error if ('is invalid' in s or 'config validity' in s)]:`
?

Reply:
@dmandyna-r7  I think this is the last comment to fix here and then we need to merge into #799

Reply:
@jschipp-r7 / @mrinehart-r7 Thanks for picking this up. I'll get that resolved shortly!
Sorry for lack of response, I was unavailable for the last couple of days.